WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -44,6 +44,35 @@ local function Show_Long_Tooltip(owner, description)
     GameTooltip:Show();
 end
 
+local function IsOverDisplay(display)
+    if not display then return nil end
+
+    local button = WeakAuras.GetDisplayButton(display)
+    return MouseIsOver(button.frame)
```

Comment:
I wanted to defer that because i thought that would be complicated.
I realised by now that designing lua code is different from C++ :laugh: